src/plugins/data_source_management/public/components/create_data_source_wizard/components/header/header.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import { EuiSpacer, EuiTitle, EuiText, EuiLink, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+
+import { FormattedMessage } from '@osd/i18n/react';
+import { DocLinksStart } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../../../opensearch_dashboards_react/public';
+import { DataSourceManagementContext } from '../../../../types';
+import { createDataSourceHeader } from '../../../text_content/text_content';
+
+export const Header = ({ docLinks }: { docLinks: DocLinksStart }) => {
+  const changeTitle = useOpenSearchDashboards<DataSourceManagementContext>().services.chrome
+    .docTitle.change;
+
+  changeTitle(createDataSourceHeader);
+
+  return (
+    <EuiFlexGroup justifyContent="spaceBetween">
+      <EuiFlexItem grow={false}>
+        <div>
+          <EuiTitle>
+            <h1 data-test-subj="createDataSourceHeader">{createDataSourceHeader}</h1>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText>
+            <p>
+              <FormattedMessage
+                id="dataSourcesManagement.createDataSource.description"
+                defaultMessage="A data source is an OpenSearch cluster endpoint (for now) to query against."
```

Comment:
I would drop the (for now) it's really ambiguous.

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2400#issuecomment-1254254646

Reply:
will address in #2434, need input from UI/UX